docs/specification/current.md

```diff
@@ -240,45 +240,66 @@ A prefetch token is a placeholder in a prefetch template that is *_replaced by i
+    "patient": "Patient?_id={{context.patient}}&_include=Patient.general-practitioner",
+    "bloodPressures": "Observation?patient={{context.patient}}&code=85354-9|http%xx/loinc.org&_sort=-date&_count=10",
+    "observationDef": "ObservationDefinition?code={{context.draftOrdersServiceRequest.code.coding}}",
+    "prescriberPr": "Practitioner?_id={{context.draftOrders.ofType(MedicationRequest).requester.reference:Practitioner}}",
+    "userPR": "PractitionerRole?_id={{userPractitionerRoleId}}&_include=PractitionerRole:practitioner"
   }
 }
 ```
 
-A prefetch template may include any of the following prefetch tokens:
-
-
-Token | Description
----|---
-`{{userPractitionerId}}` | FHIR id of the Practitioner resource corresponding to the current user. 
-`{{userPractitionerRoleId}}`|FHIR id of the PractitionerRole resource corresponding to the current user. 
-`{{userPatientId}}`|FHIR id of the Patient resource corresponding to the current user. 
-`{{userRelatedPersonId}}`|FHIR id of the RelatedPerson resource corresponding to the current user. 
-
-
-No single FHIR resource represents a user, rather Practitioner and PractitionerRole may be jointly used to represent a provider, and Patient or RelatedPerson are used to represent a patient or their proxy. Hook definitions typically define a `context.userId` field and corresponding prefetch token.
+* The 'patient' prefetch is similar to the Patient read example, but takes advantage of search to 'include' related resources.
+* The 'bloodPressures' example shows retrieving resources associated with the context patient but not otherwise related to context
+* The 'observationDef' prefetch shows the filtering based on a deep path, complex object, and by something other than id
+* The 'presciberPR' prefetch shows converting relative references to a subset of ids.
+* The 'userPR' prefetch shows the ability to filter to a subset of ids using the special legacy 'userId' naming mechanism
 
+Note:
+* Authors SHOULD NOT create prefetches that presume availability of context information not expected to be available in persisted form.  For example `MedicationRequest?_id={{context.draftOrders.id}}` would be problematic because the the draft orders aren't expected to have been persisted yet (though they might be on some clients).
 
-#### Prefetch query restrictions
+###### Prefetch query restrictions
 
 To reduce the implementation burden on CDS Clients that support CDS Services, this specification RECOMMENDS that prefetch queries only use a subset of the full functionality available in the FHIR specification. When using this subset, valid prefetch templates MUST only make use of:
 
-* _instance_ level [read](https://www.hl7.org/fhir/http.html#read) interactions (for resources with known ids such as `Patient`, `Practitioner`, or `Encounter`)
 * _type_ level [search](https://www.hl7.org/fhir/http.html#search) interactions; e.g. `patient={{context.patientId}}`
 * Resource references (e.g. `patient={{context.patientId}}`)
```

Comment:
This prohibits chaining, yes? It'd be good to annotate the examples above to say whether they are "compatible with the restricted subset".